src/components/GlobalStyle.js

```diff
@@ -9,6 +9,6 @@ export default createGlobalStyle`
     font-family:  'Helvetica Neue', sans-serif;
     font-size: 112.5%;
     line-height: 1.5;
-    align-items: center;
     font-size:12px;
```

Comment:
A space is missing after your property. Nothing critical, but it seems like prettier is not updating your CSS within your styled-component. You can either use the `.jsx`ending or set file language settings in VS Code from `JavaScript`to `JavaScript React`.